docs/ui/appui/content-control.md

```diff
@@ -1,5 +1,122 @@
-# Content Control
+# Define Content
 
-## Create
+## Overview
 
-WIP
+The content in AppUI is what is displayed in the back of your application, it is the main reason your application exists, to show, modify and interact with the content.
+
+This content must be defined and is split in 3 ideas: Control, Layout and Group
+You will use a combination of these to create the content that will be displayed in your app.
+
+## Content Control
+
+iTwin.js application typically will use a viewport to display the 2d or 3d data, and AppUI provides an IModel Viewport control to display such information. It have different feature like View Overlays, and allowing the viewer to set the viewstate and the iModel connection specifically for this viewport.
+
+You never instantiate a content control directly, it will be instantiated by the framework when it is needed. All you need to do is tell the framework that you want it with an object satisfying the ContentProps interface:
+
+```ts
+{
```

Comment:
@GerardasB I see that you have includes for these in `Frontstage.md`, but I couldnt find where they are expected to be extracted, were you planning on building that from Storybook as we discussed, or are these coming from the `test-app` ? (Or are the name placeholders only ?)

Reply:
For now only a placeholder, I was planning to add/extract these snippets from the storybook. My WIP branch https://github.com/iTwin/appui/tree/docs-snippets